Application integration tests fail in local staging environment

# Bug

## Description

Application integration tests fail in local staging environment.

## Steps to Reproduce

* `make build-debs`
* `vagrant up staging`
* `sudo su www-data -s /bin/bash`
* `cd /var/www/securedrop`
* `pytest -v tests/`

## Expected Behavior

All tests should pass.

## Actual Behavior

Integration tests fail due to selenium/Firefox WebDriver exception:

```
E               WebDriverException: Message: Can't load the profile. Profile Dir: /tmp/tmpJDEyXO If you specified a log_file in the FirefoxBinary constructor, check it for details.

/usr/local/lib/python2.7/dist-packages/selenium/webdriver/firefox/firefox_binary.py:108: WebDriverException
```
Full trace here: https://gist.github.com/emkll/e3767437a0aee4c3c40765789d267545

--- a/docs/development/testing_application_tests.rst
+++ b/docs/development/testing_application_tests.rst
@@ -48,9 +48,14 @@ Or the app-staging VM:
 .. code:: sh
 
     vagrant ssh app-staging
-    sudo su www-data -s /bin/bash
+    sudo bash
     cd /var/www/securedrop
     pytest -v tests
+    chown -R www-data /var/lib/securedrop /var/www/securedrop
+
+.. warning:: The ``chown`` is necessary because running the tests as
+             root will change ownership of some files, creating
+             problems with the source and journalist interfaces.
 
 For explanation of the difference between these machines, see
 :doc:`virtual_environments`.

